File: lista0java/src/main/java/com/mycompany/lista0java/Punkt2D.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
 */
package com.mycompany.lista0java;

/**
 *
 * @author largo
 */
public class Punkt2D {
    
    double x1;
    double y1;
    
    Punkt2D() {
        x1 = 1;
        y1 = 1;
    }

    Punkt2D(double a, double b) {
        x1 = a;
        y1 = b;

    }

    double wezX() {
        return x1;
    }

    double wezY() {
        return y1;
    }

    void wyswietl() {
        System.out.print("Punkt2D znajduje sie na nastepujacych wspolrzednych: (" + x1 + "," + y1 + ")");
    }

}

File: Lista0/Lista0/punkt2D.cpp
#include <iostream>
#include "punkt2D.h"

using namespace std;

punkt2D::punkt2D() {
    x1 = 1;
    y1 = 1;
}

punkt2D::punkt2D(double a, double b) {
    x1 = a;
    y1 = b;

}

double punkt2D::wezX() const {
    return x1;
}

double punkt2D::wezY() const {
    return y1;
}

void punkt2D::wyswietl() const {
    cout << "Punkt2D znajduje sie na nastepujacych wspolrzednych: " << x1 << "," << y1;

}
